### DISCUSSION TITLE
Lost Access to GitHub Account: Urgent Help Needed

### DISCUSSION THREAD

**Discussion by lokabhiramchintada** [community/community#144051] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi all,

I'm facing a difficult situation with my GitHub account, lokabhiram. My PC crashed recently, and unfortunately, I lost all my 2FA recovery codes and passkeys. I'm unable to access my account now.

I've tried the standard recovery methods, but they require information I no longer have. I'm reaching out to the community in hopes of finding a solution or guidance. I'm willing to provide any additional information or verification that might be needed.

Any help or advice would be greatly appreciated. Thank you for your time and consideration.

**Comment by Notorious-Ali** [comment]
same shit , hope some one can help

**Comment by Srishti44-g** [comment]
- Reach Out to GitHub Support Directly (https://support.github.com/request)
- Check for Trusted Devices or Authorized Sessions
If you were signed in on another device (like a phone or another computer), you may be able to generate new recovery codes.
- Use Linked Email
Sometimes GitHub can verify account ownership through an associated email account if you can still access it. Be sure to specify this in your support request if it’s an option.
- Enable Backup Methods in the Future
-When you regain access, consider setting up multiple backup methods (e.g., SMS, additional recovery codes stored securely) to prevent a similar issue in the future.

**Comment by Nishukvgce** [comment]
To restore access to your GitHub account, follow these steps:

1. Fill out the 2FA Account Recovery Form: Go to GitHub’s [2FA recovery page](https://github.com/settings/security#two-factor-authentication) and provide details like your username, email, and any public repositories or recent activities to verify your identity.

2. Wait for GitHub Support: GitHub will review your request, which may take several days. Be patient and check your email for updates.

3. Contact GitHub Support: If you don’t get a response, email GitHub at support@github.com with your account details for further assistance.

Once you regain access, enable multiple 2FA methods (e.g., authenticator app and hardware key) for future security.
